src/main/kotlin/com/ajaxproject/telegrambot/bot/utils/TextsUtils.kt

```diff
@@ -0,0 +1,59 @@
+package com.ajaxproject.telegrambot.bot.utils
+
+import jakarta.annotation.PostConstruct
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+import java.io.File
+import java.io.FileInputStream
+import java.io.IOException
+import java.util.*
+
+enum class Id {
+    WELCOME,
+    FUNCTIONS
+}
+
+@Component
+class TextsUtils(
+    @Value("\${text.path}")
+    val path: String,
+    @Value("\${text.fileName}")
+    val fileName: String
+) {
+    private val textsMap: MutableMap<String, Properties> = mutableMapOf()
+
+    fun getText(id: Id): String {
+        val text = textsMap[fileName]!!.getProperty(id.name)
```

Comment:
if the `textMap` doesn't contain `fileName` then you will receive NPE.

It's better use:
```kotlin
val text: String = textsMap[fileName]?.getProperty(id.name) 
            ?: throw MyCustomException("File name $fileName not found")
```